aea/cli/generate.py

```diff
@@ -20,9 +20,6 @@
 """Implementation of the 'aea generate' subcommand."""
```

Comment:
Code related to `Black` and `protoc` are now in the generator as they are needed by the generator and not just by the generate cli command.